doc.html

```diff
@@ -0,0 +1,1296 @@
+                                                </p>
+                                                <p class="info_text" style="text-align:justify">
+                                                    Once collection tools have been registered, the work flow of a benchmark script is as follows:
+                                                </p>
+                                                <div class="info_text" style="text-align:justify">
+                                                    <li>Process options (see <a href="#availablescript">Benchmark scripts options</a>).</li>
+                                                    <li>Check that the necessary prerequisites are installed and if not, install them.</li>
+                                                    <li>Iterate over some set of benchmark characteristics (e.g. pbench-fio iterates over a couple test types: read, randread and a bunch of block sizes), with each iteration doing the following:</li>
+                                                    <ul>
+                                                        <li class="subList">create a benchmark_results directory</li>
+                                                        <li class="subList">start the collection tools</li>
+                                                        <li class="subList">run the benchmark</li>
+                                                        <li class="subList">stop the collection tools</li>
+                                                        <li class="subList">postprocess the collection tools data</li>
+                                                    </ul>
+                                                </div>
+                                                <p class="info_text" style="text-align:justify">The tools are started with an invocation of pbench-start-tools like this:</p>
+                                                <p class="c_snip">pbench-start-tools --group=$group --iteration=$iteration --dir=$benchmark_tools_dir</p>
+                                                <p class="info_text" style="text-align:justify">where the group is usually "default" but can be changed to taste as described above, iteration is a benchmark-specific tag that disambiguates the separate
+                                                iterations in a run (e.g. for pbench-fio it is a combination of a count, the test type, the block size and a device name), and the benchmark_tools_dir specifies where the collection results are
+                                                going to end up (see the section for much more detail on this).</p>
+                                                <p class="info_text" style="text-align:justify">The stop invocation is exactly parallel, as is the postprocessing invocation:</p>
```

Comment:
Not clear what "exactly parallel" here is supposed to mean, vs just saying "parallel"?

Reply:
Remove "exactly" from here